src/core/generateZodSchema.test.ts

```diff
@@ -950,26 +950,26 @@ describe("generateZodSchema", () => {
            *
            * @format ipv4
            */
-          ipv4: z.string().ip({ version: "v4" }),
+          ipv4: z.string().ipv4({ version: "v4" }),
           /**
            * The hero's ipv6 address.
            *
            * @format ipv6
            */
-          ipv6: z.string().ip({ version: "v6" }),
+          ipv6: z.string().ipv6({ version: "v6" }),
```

Comment:
No more need to generate the `version`
```suggestion
          ipv4: z.string().ipv6(),
```